shelley/design-spec/delegation_design_spec.tex

```diff
@@ -74,11 +74,12 @@ \section*{List of Contributors}
 \section{Purpose}
 \label{purpose}
 
-Delegation will allow holders of ada to transfer their rights to
-participate in the proof of stake (\emph{PoS}) protocol to \emph{stake
-pools}. Stake pools are run by \emph{stake pool operators} (also called
-\emph{pool leaders}), and a person delegating to a stake pool is called
-\emph{delegator}, \emph{member}, or \emph{participant} of a stake pool.
+Delegation will allow holders of ada to transfer their rights to participate in
+the proof of stake (\emph{PoS}) protocol to \emph{stake pools}. Stake pools are
+run by \emph{stake pool operators} (sometimes also called \emph{pool leaders},
```

Comment:
Just curious: I'm just wondering in which context the term "pool leader" makes sense. What are they leading?